Fixtures/Miscellaneous/Plugins/DocCForBinaryDependency/Package.swift

```diff
@@ -0,0 +1,25 @@
+// swift-tools-version: 6.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "DemoKit",
+    products: [
+        .library(name: "DemoKit", targets: ["DemoKit"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/swiftlang/swift-docc-plugin", from: "1.0.0"),
```

Comment:
We have avoid `swift-docc-plugin` dependency here with writing the same symbolGraph call from a plugin target in the DemoKit package.

But to keep things simple, I think use `swift-docc-plugin` here is fine.

Reply:
There are certain CI jobs that can't access external dependencies, so this dependency needs to be removed.

Reply:
Fixed.